log.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 VMware, Inc. or its affiliates. All Rights Reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package amqp091
+
+import (
+	"log"
+	"os"
+)
+
+var ErrLog = log.New(os.Stderr, "[ERROR] amqp091-go: ", log.LstdFlags|log.Lshortfile)
+var WarnLog = log.New(os.Stderr, "[WARNING] amqp091-go: ", log.LstdFlags|log.Lshortfile)
```

Comment:
I strongly recommend to specify a simple interface with a Printf-style method instead of using the log.Logger.

Using log.Logger makes difficult for user to use his custom logger instead.

Reply:
I agree.
@lukebakken here is an example: https://github.com/ansd/lastpass-go/blob/main/log.go define the interface and here is how to use it: https://github.com/ansd/lastpass-go/blob/825ee68cdb61d29d174ddc9c0831b5266b988cda/examples/logging/example.go#L27-L33

Reply:
So, it's the user / application that decides what logger to use, not the library.